server/routes/imageUpload.js

```diff
@@ -0,0 +1,8 @@
+const express = require("express");
+const router = express.Router();
+const protect = require("../middleware/auth");
+const { uploadProfilePic } = require("../controllers/imageUpload");
+
+router.route("/profilepic").post(protect, uploadProfilePic);
```

Comment:
```suggestion
router.route("/profile-pic").post(protect, uploadProfilePic);
```

Please use hyphenated case for route names :)

Reply:
sure 👍